xapian-letor/configure.ac

```diff
@@ -565,7 +629,19 @@ AC_CONFIG_FILES([
  Makefile
  tests/Makefile
  docs/Makefile
+ cmake/xapianletor-config.cmake
+ cmake/xapianletor-config-version.cmake
  ])
 AC_CONFIG_FILES([makemanpage], [chmod +x makemanpage])
 AC_CONFIG_FILES([tests/runtest], [chmod +x tests/runtest])
+AC_CONFIG_FILES([xapianletor-config], [chmod +x xapianletor-config])
 AC_OUTPUT
+
+dnl There are no files generated by AC_OUTPUT in the following directories
+dnl and we need to ensure they exist so that the rest of configure or make
+dnl won't fail because they don't exist when srcdir != builddir.
+if test yes = "$vpath_build" ; then
+  for dir in include/xapian-letor ; do
+    AS_MKDIR_P(["$dir"])
+  done
+fi
```

Comment:
Didn't James already deal with this issue a different way in efaaad5e6eeeff0a85a89da4fe63ca960c149836?

In core we need to create `include/xapian` here because we then go on to generate `include/xapian/version.h`, but there's nothing generated in `include/xapian-letor` at configure-time.

Reply:
Indeed he did sorry fixed in a fixup commit.